app/assets/javascripts/timeline.js

```diff
@@ -209,6 +224,17 @@ samson.controller("TimelineCtrl", function($scope, $window, $timeout, Deploys, S
   $scope.timelineDeploys = Deploys;
   $scope.deploys = Deploys.entries;
 
+
+  if (!$scope.selectedTimeFormat) {
+      $scope.selectedTimeFormat = document.getElementById("default_time_format").value;
+      $scope.timeFormat = $scope.selectedTimeFormat;
+  }
+
+
+  $scope.changeTime = function() {
+    $scope.selectedTimeFormat = $scope.timeFormat;
```

Comment:
feels like this should be `global_scope. selectedTimeFormat  = $scope.timeFormat` and not assigning on itself ... then we could simply use timeFormat everywhere ... but no idea how this angular stuff works :D